### PR TITLE
fix: move experiment SQL error

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
[ET-96]

## Test Plan
- Check if move experiment API works in any cases
  - e.g: 
    - with `filters` field
    - with RBAC
    - with `excludedExperimentIds` field
    - with single/multiple experiments
    - etc...

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

[ET-96]: https://hpe-aiatscale.atlassian.net/browse/ET-96?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: master/internal/experiment/bulk_action.go
[the remark this answers is not in the document]
Comment (corban-beaird) [reply master/internal/experiment/bulk_action.go:1]: Good catch, I'm surprised we don't have automated testing around this endpoint; I'll make sure to add it to the list of items for improving coverage

**Review by corban-beaird — APPROVED** [review]
Thanks for fixing this!

File: master/internal/api_experiment_intg_test.go [hunk 1]
```diff
@@ -292,6 +292,139 @@ func TestMoveExperimentRunID(t *testing.T) {
...
+			Filters:              nil,
+		})
+		for _, v := range result.Results {
+			require.Equal(t, v.Error, "")
+		}
+		require.Equal(t, len(result.Results), 2)
+		require.NoError(t, err)
+	})
+
+	t.Run("Move experiements with filters", func(t *testing.T) {
```
Comment (corban-beaird) [inline master/internal/api_experiment_intg_test.go:328]: nit: I believe that there is a typo on experiments

Comment (keita-determined) [reply master/internal/api_experiment_intg_test.go]: fixed all typos

**Review by corban-beaird — APPROVED** [review]
Looks great! Thanks for adding testing and resolving the SQL error with the endpoint!

**Pull request merged**